### PR TITLE
DEVSUPP-224: Add secondary menu to mobile nav

### PR DESCRIPTION
**Pull request by yepzdk**
https://jira.itkdev.dk/browse/DEVSUPP-224
Include secondary menu in mobile nav

### CODE REVIEW

File: web/themes/custom/hoeringsportal/templates/navigation/menu--secondary-navigation.html.twig [hunk 1]
```diff
@@ -30,13 +30,13 @@
...
     {% if menu_level == 0 %}
-      <ul{{ attributes.addClass('navbar-nav is-secondary-navigation mr-auto float-right') }}>
+      <ul{{ attributes.addClass('navbar-nav is-secondary-navigation mr-lg-auto float-lg-right') }}>
     {% else %}
       <ul>
     {% endif %}
     {% for item in items %}
-      <li{{ item.attributes }} class="nav-item--secondary-navigation {{ item.in_active_trail ? 'active' : '' }}">
-        {{ link(item.title, item.url, { 'class':['nav-link', 'has-border']} ) }}
+      <li{{ item.attributes }} class="nav-item nav-item--secondary-navigation {{ item.in_active_trail ? 'active' : '' }}">
```
Comment (rimi-itk) [inline web/themes/custom/hoeringsportal/templates/navigation/menu--secondary-navigation.html.twig:38]: Not directly related to the changes in this PR, but `{{ item.in_active_trail ? 'active' : '' }}` can be written as just `{{ item.in_active_trail ? 'active' }}` (cf. https://twig.symfony.com/doc/3.x/templates.html#other-operators).

File: web/themes/custom/hoeringsportal/templates/layout/region--header.html.twig [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * Available variables:
+ * - content: The content for this region, typically blocks.
+ * - attributes: HTML attributes for the region div.
+ * - region: The name of the region variable as defined in the theme's
+ *   .info.yml file.
+ *
+ * @see template_preprocess_region()
+ */
+#}
+{% if content %}
```
Comment (rimi-itk) [inline web/themes/custom/hoeringsportal/templates/layout/region--header.html.twig:15]: This should probable be changed to `{% if elements %}` (we don't use `content` elsewhere in the template) or be removed.

It seems weird to me that we don't have a wrapper element (like `<div class="header">…</div>`, say) around the content in this template, but that's just a personal opinion.

File: web/themes/custom/hoeringsportal/templates/navigation/menu--secondary-navigation.html.twig (hunk 1 above)
Comment (yepzdk) [reply web/themes/custom/hoeringsportal/templates/navigation/menu--secondary-navigation.html.twig]: Thanks for the suggestion. It has been addressed in this commit: https://github.com/itk-dev/hoeringsportal/pull/67/commits/e74c6688664309f39336a0212894820868599eb5

File: web/themes/custom/hoeringsportal/templates/layout/region--header.html.twig (hunk 2 above)
Comment (yepzdk) [reply web/themes/custom/hoeringsportal/templates/layout/region--header.html.twig]: That makes perfectly sense. Removed `content` and moved the div container from the parent template, into this template, with this commit: https://github.com/itk-dev/hoeringsportal/pull/67/commits/689e384e53c1ce1b6472f21ebd468cbc06c348c9

**Pull request merged**